bcipy/helpers/stimuli.py

```diff
@@ -273,6 +303,48 @@ def calibration_inquiry_generator(
     return InquirySchedule(samples, times, colors)
 
 
+def distributed_target_positions(stim_number: int, stim_length: int, nontarget_inquiries: int) -> list:
```

Comment:
I like how the calculation of target indexes is split off into a separate function. I think the implementation can probably be simplified using some of the techniques above. You should also be explicit in the documentation about what value is used when the target should not be shown (my preference is `None` as described above).